packages/mobile/src/account/InviteReview.tsx

```diff
@@ -137,28 +137,23 @@ export class InviteReview extends React.Component<Props, State> {
           onPress={this.onInviteSMS}
           text={t('inviteWithSMS')}
           accessibilityLabel={t('inviteWithSMS')}
-          standard={false}
           type={BtnTypes.TERTIARY}
           disabled={inviteInProgress}
-        >
-          <SMSLogo />
-        </GethAwareButton>
+        />
+        <SMSLogo />
```

Comment:
The logo should be part of the button.

Reply:
Good catch. I actually did that on purpose because the v2 buttons don't support child elements. I didn't want to create a new component just for this screen because this entire flow has been deprecated. I will probably completely delete this a following PR.